### ISSUE TITLE
"Not allowed to load local resource" when adding a script from node modules

### ISSUE THREAD

**Issue by Jeneko** [webdiscus/pug-plugin#65]
Good day! I'm trying to add **Bootstrap** to my project when using **pug-plugin**. As recipes said I do:

```
script(src=require('bootstrap/dist/js/bootstrap.bundle.js'))
```

Then I start **webpack-dev-server**. And everything goes good until I make a change in any **.scss** or **.js** file of the project and then save the file. After page autoreload I get this error in browser console:

![error-image](https://user-images.githubusercontent.com/16610240/222921683-bd8c495e-b8a1-40c0-bf20-a609417d0ae0.png)

Strangely enough, if I make a change in **.pug** file the error doesn't appear. And if I put **bootstrap.bundle.js** out of **/node_modules** somewhere into **/src** and load it from there the error also does not appear.

As a test I add **Bootstrap** in the standard **pug-plugin** example **"simple-multipage"**. [Here is the repo](https://github.com/Jeneko/pug-plugin-issue)

**Comment by webdiscus** [comment]
Hello @Jeneko,

Thanks for reporting the issue, I'll fix it.

**Comment by Jeneko** [comment]
@webdiscus, thank you very much!

**Comment by webdiscus** [comment]
@Jeneko 

the issuer is fixed in v4.9.7.

> **Note**
>
> You can load CSS and JS from a node module using only module name without a path to a file in module.
> The plugin automatically detect the  default CSS and default JS defined `package.json` of the module.
>
> For example:
> `link(href=require("bootstrap") rel="stylesheet")` => resolves the `bootstrap/dist/css/bootstrap.css`
> `script(src=require("bootstrap"))` => resolves the `bootstrap/dist/js/bootstrap.js`

You can see the working example [watch-dependencies-bootstrap](https://github.com/webdiscus/pug-plugin/tree/master/test/manual/watch-dependencies-bootstrap).

**Comment by Jeneko** [comment]
@webdiscus 
Thanks a lot for the quick fix! Everything works perfect!
